lib/signal_tower.ex

```diff
@@ -45,4 +40,16 @@ defmodule SignalTower do
     Logger.info("SignalTower started")
     ret
   end
+
+  @impl Application
+  def stop(_) do
```

Comment:
I liked the `_state` in the previous version, because it gives me an idea about the variable content/type

Reply:
Ok, I will bring it back ;-)